src/pages/workspace/WorkspaceInviteMessagePage.js

```diff
@@ -82,7 +82,7 @@ class WorkspaceInviteMessagePage extends React.Component {
 
     componentDidMount() {
         if (_.isEmpty(this.props.invitedEmailsToAccountIDsDraft)) {
-            Navigation.goBack(ROUTES.getWorkspaceInviteRoute(this.props.route.params.policyID), true);
+            return Navigation.goBack(ROUTES.getWorkspaceInviteRoute(this.props.route.params.policyID), true);
```

Comment:
I meant
```js
Navigation.goBack(...);
return;
```
Not much difference I guess but that looks more readable and it's clearer that we are doing this for an early return situation

Reply:
Ok, updated.